src/python/genny/cedar.py

```diff
@@ -198,6 +201,10 @@ def split_into_actor_csv_files(data_reader, out_dir):
 
         cur_out_csv.writerow(line)
 
+        counter += 1
+        if counter % 1e6 == 0:
+            print('Parsed {} metrics'.format(counter))
```

Comment:
[note] You can also use f-strings as long as we're willing to set a floor of Python 3.6.

https://docs.python.org/3/reference/lexical_analysis.html#f-strings